library/private/interactor_impl.h

```diff
@@ -38,6 +38,11 @@ class interactor_impl : public interactor
   interactor& setKeyPressCallBack(std::function<bool(int, std::string)> callBack) override;
   interactor& setDropFilesCallBack(std::function<bool(std::vector<std::string>)> callBack) override;
 
+  interactor& addCommandCallback(
```

Comment:
I'm wondering if we shouldn't have a `const std::vector<std::string>&` as argument of the callback.  
That way, we can handle splitting arguments at the framework level, and not reimplement it in every callback.  
I think parsing might become more complicated in the future (like handling escaped spaces, handling several consecutive spaces, quoted arguments, ...)

Reply:
> not reimplement it in every callback.

Only a few have it

The problem I have with this is that `set option.vector 1.3, 1.4, 1.5` will be split and then need to be recombined before being given to `options::set`.

Reply:
There is some benefit though, lets discuss pros/cons

Reply:
I see you pushed this change. Should I consider we agree and close the discussion?

Reply:
indeed